archive/analyze-workflow-structure.mjs

```
@@ -0,0 +1,191 @@
+// Analyze connections
+let connectionCount = 0;
+const nodeConnections = {};
+
+for (const [sourceNode, outputs] of Object.entries(connections)) {
+  nodeConnections[sourceNode] = {
+    outputs: [],
+    totalOutputs: 0
+  };
+  
+  if (outputs.main && outputs.main[0]) {
+    outputs.main[0].forEach(connection => {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)